src/libostree/ostree-sysroot.c

```diff
@@ -1338,7 +1338,7 @@ ostree_sysroot_try_lock (OstreeSysroot         *self,
         }
       else
         {
-          g_propagate_error (error, local_error);
+          g_propagate_error (error, g_steal_pointer (&local_error));
```

Comment:
This is a pattern I think we should propagate to every other use of `g_propagate_error()` to avoid other errors like this.